pkgs/development/node-packages/default.nix

```diff
@@ -382,16 +378,16 @@ let
       }
     );
 
-    ssb-server = super.ssb-server.override {
+    ssb-server = super.ssb-server.override (oldAttrs: {
       buildInputs = [ pkgs.automake pkgs.autoconf self.node-gyp-build ];
-      meta.broken = since "10";
-    };
+      meta = oldAttrs.meta // { broken = since "10"; };
+    });
 
-    stf = super.stf.override {
-      meta.broken = since "10";
-    };
+    stf = super.stf.override (oldAttrs: {
+      meta = oldAttrs.meta // { broken = since "10"; };
+    });
 
-    tailwindcss = super.tailwindcss.overrideAttrs (oldAttrs: {
+    tailwindcss = super.tailwindcss.override {
```

Comment:
Really unsure about this change.

Reply:
Honestly I don't understand the subtleties when it comes to using `override` vs `overrideAttrs`, e.g., I was surprised to see `override` being passed a function as an argument, as opposed to an attribute set, in this file.

As far as I can tell this change doesn't produce any meaningful difference to the derivation. `nixpkgs-review` doesn't notice any change that would have it try and rebuild the package, and the package builds and runs fine on my system.